Jenkinsfile

```diff
@@ -18,58 +18,67 @@
+            if (utils.isCI()) {
 
-        } else if (utils.isCD()) {
+                echo 'CI pipeline'
 
-            //First we need to check that master is
-            //stable and all tests are working properly
-            //before generating tags and pushing it to github
+                sh "mvn clean install"
 
-            def stagedProject
+            } else if (utils.isCD()) {
 
-            //These stage(), release(stagedProject) and
-            //updateDownstreamDependencies(stagedProject)
-            //are coming form release.groovy in same repo
+                // First we need to check that master is
+                // stable and all tests are working properly
+                // before generating tags and pushing it to github
 
-            stage('Stage') {
+                sh "mvn clean -B"
 
-                // This will generate new tag, push them to github,
-                // create the new release branch, build and run test
+                sh "mvn clean install"
 
-                stagedProject = pipeline.stage()
-            }
+                def stagedProject
 
-            stage('Promote') {
+                // These stage(), release(stagedProject) and
+                // updateDownstreamDependencies(stagedProject)
+                // are coming form release.groovy in same repo
 
-                //This will release the project and update on
-                //maven central
+                stage('Stage') {
 
-                pipeline.release(stagedProject)
-            }
+                    // This will generate new tag, push them to github,
+                    // create the new release branch, build and run test
+
+                    stagedProject = pipeline.stage()
+                }
 
-            // Disabled for now as it probably doesn't work because of the different directory structure
-            // with a dedicated doc-module
-            //stage 'Website'
-            //pipeline.website(stagedProject)
+                stage('Promote') {
 
-            stage('Update downstream dependencies') {
+                    // This will release the project and update on
+                    // maven central
 
-                //This will update the version of fmp
-                //in the pom.xml of some downstream repos like
-                //fabric8-services, quickstart etc.
-                pipeline.updateDownstreamDependencies(stagedProject)
+                    pipeline.release(stagedProject)
+                }
+
+                // Disabled for now as it probably doesn't work because of the different directory structure
+                // with a dedicated doc-module
+                // stage 'Website'
+                // pipeline.website(stagedProject)
+
+                stage('Update downstream dependencies') {
+
+                    // This will update the version of fmp
+                    // in the pom.xml of some downstream repos like
+                    // fabric8-services, quickstart etc.
```

Comment:
Does it update the quickstart projects?

Reply:
Here is the list https://github.com/fabric8io/fabric8-maven-plugin/blob/master/release.groovy#L50

Reply:
This test is working right now also, but to run it in Jenkins file this change is required. If we do this change, then will this Jenkins file will work fine.